src/utils/logger.ts

```diff
@@ -94,12 +94,13 @@ export function logWarn(...args: any[]): void {
 export function logError(...args: any[]): void {
     localLog(LogType.Error, ...args);
 }
-
+/* istanbul ignore next */
```

Comment:
Hva betyr istanbul?

Reply:
Tror det ignorerer linjer for test coverage?

Reply:
da ignorerer man tester i sammendraget ;) er noe jest greier ;)

Reply:
Ah 👍